packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -527,6 +527,12 @@ export interface UserPoolProps {
    */
   readonly mfa?: Mfa;
 
+  /**
+   * A string representing the MFA message.
+   * @default - Omitted by default.
```

Comment:
Is this the correct default?

I assume that turning on SMS based MFA but not specifying a message would mean that users in the user pool will receive some default SMS message, no?

If so, that should be the documented default.